WORKSPACE

```diff
@@ -141,12 +141,15 @@ load("@rules_proto_grpc//java:repositories.bzl", rules_proto_grpc_java_repos = "
 
 rules_proto_grpc_java_repos()
 
-load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
```

Comment:
Why remove this rule set?

Reply:
Buildifier flagged it as a lint error in my editor. It's identical to the line below. Loading twice has no effect.

Reply:
Ah, I see now.